committeeoversightapp/templates/hearing_delete.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html' %}
```

Comment:
This file was just renamed `hearing_delete.html` from `delete.html`. The same applies to `hearing_edit.html` and `hearing_create.html`